### DISCUSSION TITLE
cropping an steam

### DISCUSSION THREAD

**Discussion by kcsaby** [blakeblackshear/frigate#14723] [Ask A Question]
Is it possible to use an option in ffmeg to crop a video steam so that it can be restreamed.
Have a steam that I would like to crop out to get a better view of that area 
something like "ffmpeg -i inputsteam -vf "crop=640:480" output.mp4"
have seen examples using ffmeg to do this and to set up go2rtc to exec ffmeg command but how would you format the command to take an incoming rtsp feed and then restream it

Thanks

**Comment by NickM-27** [comment]
yes, this would be done using go2rtc, go2rtc github has examples of this
